app/dashboard/views.py

```diff
@@ -2161,7 +2161,7 @@ def user_card(request, handle):
     except (ProfileNotFoundException, ProfileHiddenException):
         raise Http404
 
-    if request.user.is_authenticated:
+    if request.user.is_authenticated and request.user.profile:
```

Comment:
Is this a symptom of a different problem? Why do we have authenticated users without `Profiles`?

Reply:
** As discussed we will get this in now and open a new ticket to investigate the cause